packages/concerto-core/test/introspect/identifieddeclaration.js

```diff
@@ -52,7 +52,7 @@ asset Order {
             mm.addModelFile( `
 namespace test
 
-asset Order {
+asset Order identified {
```

Comment:
To be discussed I think. This now allows assets to not be identifiable - which seems odd.

Reply:
I just never know which of those are meant to be `identified` as a built-in thing or as a user decision. Is it just assets? I'm a little unclear there is a benefit those distinctions, but it may be because I forgot earlier discussions.

I'll play a bit by adding `identified` to assets and report.